src/Enterspeed.Source.SitecoreCms.V8/Services/EnterspeedConfigurationService.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Enterspeed.Source.SitecoreCms.V8.Exceptions;
+using Enterspeed.Source.SitecoreCms.V8.Models.Configuration;
+using Sitecore.Abstractions;
+using Sitecore.Configuration;
+using Sitecore.Data.Fields;
+using Sitecore.Data.Items;
+using Sitecore.Globalization;
+using Sitecore.Links;
+using Sitecore.Sites;
+using Sitecore.Web;
+using Version = Sitecore.Data.Version;
+
+namespace Enterspeed.Source.SitecoreCms.V8.Services
+{
+    public class EnterspeedConfigurationService : IEnterspeedConfigurationService
+    {
+        private readonly BaseLanguageManager _languageManager;
+        private readonly BaseItemManager _itemManager;
+        private readonly BaseLinkManager _linkManager;
+        private readonly BaseFactory _factory;
+        private readonly BaseSiteContextFactory _siteContextFactory;
+
+        private List<EnterspeedSitecoreConfiguration> _configuration;
+        private Guid _configurationRevisionId = Guid.Empty;
+
+        private DateTime _lastUpdatedDate = DateTime.MinValue;
+
+        public EnterspeedConfigurationService(
+            BaseLanguageManager languageManager,
+            BaseItemManager itemManager,
+            BaseLinkManager linkManager,
+            BaseFactory factory,
+            BaseSiteContextFactory siteContextFactory)
+        {
+            _languageManager = languageManager;
+            _itemManager = itemManager;
+            _linkManager = linkManager;
+            _factory = factory;
+            _siteContextFactory = siteContextFactory;
+        }
+
+        public List<EnterspeedSitecoreConfiguration> GetConfiguration()
+        {
+            Item enterspeedConfigurationItem = _itemManager.GetItem(EnterspeedIDs.Items.EnterspeedConfigurationID, Language.Parse("en"), Version.Latest, _factory.GetDatabase("web"));
+            if (enterspeedConfigurationItem == null || enterspeedConfigurationItem.Versions.Count == 0)
+            {
+                return new List<EnterspeedSitecoreConfiguration>();
+            }
+
+            //string enabled = enterspeedConfigurationItem[EnterspeedIDs.Fields.EnterspeedEnabledFieldID] ?? string.Empty;
```

Comment:
If this commented out part is irrelevant, can you please remove it?

Reply:
@elglogins I have pushed the changes to the branch.